depmapomics/vcf.py

```diff
@@ -418,6 +420,10 @@ def improve(
     print("replacing empty characters:")
     vcf = vcf.replace(replace_empty)
 
+    print("re-annotating CIVIC using static dataframe:")
+    vcf = civic_df.merge(vcf, on=['chrom','pos', 'ref', 'alt'], how='right')
```

Comment:
I think this works and I don't see any problems, but just wanted to share a few common conventions:

When adding things to a dataframe, it's more common to have the main table and add it onto the "left". I think this is because people tend to read left-to-right so it's easier to imagine tacking things onto the end (on the right)

```
civic_df.merge(vcf, on=['chrom','pos', 'ref', 'alt'], how='right')
````

changes to 

```
vcf = vcf.merge(civic_df, on=['chrom','pos', 'ref', 'alt'], how='left')
```

Another minor tip: I'd actually prepare civic_df to have the names you want before merging, and drop the columns you don't want.

Again, these are purely stylistic choices that I make because (IMHO) I think it makes it easier to follow what's happening.

If this works, no need to change it at this time.

Reply:
Agreed! I won't have time to change it for this quarter but it does makes more sense to have them added at the end. I will add a task and address it later